src/Operation/Normalize.php

```diff
@@ -24,15 +24,15 @@ final class Normalize extends AbstractOperation
     /**
      * @pure
      *
-     * @return Closure(Iterator<TKey, T>): Generator<int, T>
+     * @return Closure(Iterator<TKey, T>): Generator<int, T, mixed, void>
      */
     public function __invoke(): Closure
     {
         return
             /**
              * @param Iterator<TKey, T> $iterator
              *
-             * @return Generator<int, T>
+             * @return Generator<int, T, mixed, void>
```

Comment:
does it add to the type information to add the other Generator generics, or were they already inferred as those types?

Reply:
Hi Alex,

I don't understand your question, can you elaborate?

To give you a bit of context, this was spotted by PHPStan when using (see [this tweet](https://twitter.com/OndrejMirtes/status/1437291613336784902)):

```yaml
parameters:
    checkExplicitMixed: true
```

This is why I added some of these things here and there.

Reply:
ok, sounds good! Should we add that in the PHPStan config in a separate PR?

Reply:
No, not yet, I'll wait the next release of Phpstan before making those changes. I even think that it will be enabled by default in the new level 9 :)